response_operations_ui/controllers/collection_exercise_controllers.py

```diff
@@ -4,6 +4,7 @@
 from flask import current_app as app
 from requests.exceptions import HTTPError
 from structlog import wrap_logger
+import json
```

Comment:
This should be in the top block (where logging is) as json is in the standard library